### DISCUSSION TITLE
Can this plugin send all notes to existing `mkdocs` repo without setting frontmatter

### DISCUSSION THREAD

**Discussion by andynameistaken** [ObsidianPublisher/obsidian-github-publisher#177] [Q&A — Plugin]
Is it possible to send all notes to `docs` folder of preexisting repository and trigger

`mkdocs gh-deploy` without requiring any settings in the fronmatter (I always want to add all files)?

**Comment by Mara-Li** [comment]
Probably something automatically with templater.

Or just use another plugin with git
